docs/data/pages.ts

```diff
@@ -15,7 +15,19 @@ const pages = [
       { pathname: '/x/react-data-grid/getting-started' },
       { pathname: '/x/react-data-grid/migration-v4', title: 'Migration from v4' },
       { pathname: '/x/react-data-grid/layout' },
-      { pathname: '/x/react-data-grid/columns' },
+      {
+        pathname: '/x/react-data-grid/columns',
```

Comment:
`scopePathnames` is missing. Otherwise, the sidebar menu is always closed when viewing a child page.

Reply:
Same for `group-pivot`
We did not nest routes in the URL, so our current sidebar menu expansion approach does not work correctly.

AG-Grid does not nest either https://www.ag-grid.com/javascript-data-grid/filter-date/
And I don't think we should.

But in that case we need to update the behavior of the side bar so open when the current page is a children

Reply:
Why `scopePathnames` was not added here? Although the URLs are not nested, the menu items are. The "Group & Pivot" group has `scopePathnames`.

Reply:
I'm not saying that the parent menu should close, I did not understand the goal of `scopePathnames` correctly

Reply:
Defining `scopePathnames` is simpler than to build a logic to iterate through all parents and children to find the current page. If we're going to have more groups in the sidebar might make sense to implement the automatic approach.